### ISSUE TITLE
icetray-inspect -  on OSX hdfwriter is included as an inspectable project, when it is not built (Trac #1605)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1597]
`hdfwriter` doesn't build for me because I don't have the HDF5 libs installed.

`icetray-inspect` still tries to do it's thing though...

```text
hdfwriter
Error cant load 'hdfwriter': dlopen(/Users/nega/i3/combo/build/lib/icecube/hdfwriter.so, 2): image not found
HiveSplitter
```

Migrated from https://code.icecube.wisc.edu/ticket/1605
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:42", 
    "description": "`hdfwriter` doesn't build for me because I don't have the HDF5 libs installed.\n\n`icetray-inspect` still tries to do it's thing though...\n\n{{{\nhdfwriter\nError cant load 'hdfwriter': dlopen(/Users/nega/i3/combo/build/lib/icecube/hdfwriter.so, 2): image not found\nHiveSplitter\n}}}\n", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067102580394", 
    "component": "infrastructure", 
    "summary": "icetray-inspect -  on OSX hdfwriter is included as an inspectable project, when it is not built", 
    "priority": "major", 
    "keywords": "documentation", 
    "time": "2016-03-28T17:38:03", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160420T18:52:16: olivas changed milestone from "March 2016" to "April 2016"

**Comment by icecube-trac** [comment]
Trac update at 20160506T12:13:18: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "new" to "closed"
* kjmeagher commented:

> In changeset 2568:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2568"
> redo the way icetray-inspect handles available libs, cmake writes a zero length file to `${I3_BUILD}/docs/no_inspect/` for projects which can't be inspected and icetray-inspect checks there for projects to skip
> fixes #1596,#1597 and #1309
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T20:46:33: nwhitehorn commented:

> In changeset 168734:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168734"
> redo the way icetray-inspect handles available libs, cmake writes a zero length file to `${I3_BUILD}/docs/no_inspect/` for projects which can't be inspected and icetray-inspect checks there for projects to skip
> fixes #1596,#1597 and #1309
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:17:55: nwhitehorn commented:

> In changeset 170271:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170271"
> redo the way icetray-inspect handles available libs, cmake writes a zero length file to `${I3_BUILD}/docs/no_inspect/` for projects which can't be inspected and icetray-inspect checks there for projects to skip
> fixes #1596,#1597 and #1309
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:42: 
* olivas commented:

> Milestone April 2016 deleted


* olivas changed milestone from "April 2016" to ""
